src/lite-psm-halt/SingleLitePsmHaltSpell.sol

```diff
@@ -19,8 +19,9 @@ import {DssEmergencySpell} from "../DssEmergencySpell.sol";
 
 enum Flow {
     SELL, // Halt only selling gems
-    BUY,  // Halt only buying gems
-    BOTH  // Halt both
+    BUY, // Halt only buying gems
+    BOTH // Halt both
+
```

Comment:
LOL this bug in the formatter still exists.
```suggestion
```

Reply:
fixed in [chore: fix formatting](https://github.com/makerdao/dss-emergency-spells/pull/2/commits/0af25558207800836052ff913a365ba2a37e7f66)